Automattic-Tracks-iOS/Crash Logging/SentryEventSerializer.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Sentry
+
+struct SentryEventSerializer {
+
+    private let dsn: String
+    private let jsonEncoder: JSONEncoder
+
+    var events = [String: Event]()
+
+    init(dsn: String) {
+        self.dsn = dsn
+        self.jsonEncoder = JSONEncoder()
+        self.jsonEncoder.dateEncodingStrategy = .iso8601
```

Comment:
The pod validation (`pod lib lint --allow-warnings`) fails because of strategy case is only available from macOS 10.12.

The project and `Podfile` use 10.12 as the minimum deployment target, but the `.podspec` is declaring `10.11`, so `pod lib lint` builds with that one.

Reply:
Ah nice – thanks for pointing this out!!